modules/installation-configuration-parameters.adoc

```diff
@@ -1831,6 +1831,62 @@ Additional Nutanix configuration parameters are described in the following table
+|String
+
+|`controlPlane.platform.nutanix.categories.value`
+|The value of a prism category key-value pair to apply to control plane VMs. This parameter must be accompanied by the `key` parameter, and both `key` and `value` parameters must exist in Prism Central.
+|String
+
+|`controlPlane.platform.nutanix.project.type`
+|The type of identifier you use to select a project for control plane VMs.  Projects define logical groups of user roles for managing permissions, networks and other parameters. For more information on projects, see link:https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-vpc_2022_9:ssp-app-mgmt-project-env-c.html[Projects Overview].
+|`name` or `uuid`
+
+|`controlPlane.platform.nutanix.project.name` or `controlPlane.platform.nutanix.project.uuid`
+|The name or UUID of a project with which control plane VMs are associated. This parameter must be accompanied by the `type` parameter.
+|String
+
+|`platform.nutanix.defaultMachinePlatform.categories.key`
+|The name of a prism category key to apply to all VMs. This parameter must be accompanied by the `value` parameter, and both `key` and `value` parameters must exist in Prism Central. For more information on categories, see link:https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-vpc_2022_6:ssp-ssp-categories-manage-pc-c.html[Category management].
+|String
+
+|`platform.nutanix.defaultMachinePlatform.categories.value`
+|The value of a prism category key-value pair to apply to all VMs. This parameter must be accompanied by the `key` parameter, and both `key` and `value` parameters must exist in Prism Central.
+|String
+
+|`platform.nutanix.defaultMachinePlatform.project.type`
+|The type of identifier you use to select a project for all VMs. Projects define logical groups of user roles for managing permissions, networks and other parameters. For more information on projects, see link:https://portal.nutanix.com/page/documents/details?targetId=Prism-Central-Guide-vpc_2022_9:ssp-app-mgmt-project-env-c.html[Projects Overview].
+|`name` or `uuid`.
+
+|`platform.nutanix.defaultMachinePlatform.project.name` or `platform.nutanix.defaultMachinePlatform.project.uuid`
+|The name or UUID of a project with which all VMs are associated. This parameter must be accompanied by the `type` parameter.
+|String
+
+|`platform.nutanix.defaultMachinePlatform.bootType`
+|The boot type for all machines. You must use `Legacy` boot type in {product-title} {product-version}. If you use `SecureBoot`, `UEFI` will also be enabled. For more information on boot types, see link:https://portal.nutanix.com/page/documents/kbs/details?targetId=kA07V000000H3K9SAK[Understanding UEFI, Secure Boot, and TPM in the Virtualized Environment].
```

Comment:
s/Legacy/the Legacy

> If you use `SecureBoot`,
The preceding sentence says I must use `Legacy`. Which is correct?

Reply:
Good catch, I meant to remove that sentence. Thanks